spring-cloud-contract-tools/spring-cloud-contract-converters/src/main/groovy/org/springframework/cloud/contract/verifier/converter/RecursiveFilesConverter.groovy

```diff
@@ -43,27 +43,40 @@ import org.springframework.cloud.contract.verifier.wiremock.DslToWireMockClientC
 class RecursiveFilesConverter {
 
 	private final StubGeneratorProvider holder
-	private final ContractVerifierConfigProperties props
 	private final File outMappingsDir
+	private final File contractsDslDir
+	private final List<String> excludedFiles
+	private final String includedContracts
+	private final boolean excludeBuildFolders
 
+	// Use constructor without ContractVerifierConfigProperties
+	@Deprecated
```

Comment:
Decided to keep my changes here to only pass what is actually needed, so deprecating these old constructors.